### DISCUSSION TITLE
Hat das Firmware Update funktioneirt?

### DISCUSSION THREAD

**Discussion by Butz443** [tbnobody/OpenDTU#1039] [Q&A]
Hi,

eben kam mein OpenDTU an. 
Direkt in Betrieb genommen und die FW aktualisiert. 
Firmware (OPENDTU-GENERIC.bin) heruntergeladen und auf dem OpenDTU hochgeladen.
Hat alles soweit funktioniert, danach kam die Meldung, dass es wohl einen auto. Neustart gibt.
1-2 Stunden gewartet. 

Anscheinend nichts passiert:

<img width="1049" alt="Bildschirmfoto 2023-06-15 um 14 58 15" src="https://github.com/tbnobody/OpenDTU/assets/136714546/0d05128d-7020-4e81-9172-7d608d43d14d">

Das ganze nochmals durchgeführt, gewartet, manueller Neustarts.
Bringt leider nichts. Ich bekomme immer gesagt, dass eine neue Version verfügbar wäre..

Was mache ich falsch?

**Comment by Solarteur** [comment]
Habe bei mir gerade auf v23.6.14 aktualisiert - keine Probleme. Der Neustart erfolgte wie immer "augenblicklich".

Was heißt denn "eben kam mein OpenDTU an. Direkt in Betrieb genommen"? War schon unter v23.6.9 ein WR eingerichtet und es wurden Daten übertragen?
